### ISSUE TITLE
[AXE-CORE]: UL and OL must only directly contain LI, SCRIPT, or TEMPLATE elements

### ISSUE THREAD

**Issue by andreahewitt-odd** [department-of-veterans-affairs/va.gov-team#1064]
## Issue
The list items have a `role="presentation"` that is causing issues with Axe. Screenshot below.

## Issue Type
* Defect

## Acceptance Criteria
* As a user, I want the list items to have proper markup so they are useful to screen reader users and do not cause axe validation errors

## Environment
* MacOS High Sierra (one version below current, Mojave)
* Chrome latest
* https://staging.va.gov/track-claims/your-claims/600141546/status
* login: ID.me
* user: Test user 228

## Steps to Recreate
1. Log into staging.va.gov with ID.me (test user listed above in Environment)
2. Enter `https://staging.va.gov/track-claims/your-claims/600141546/status` in browser
3. Open Developer Tools and click the axe `ANALYZE` button (Assume [aXe](https://www.deque.com/axe) browser plugin is installed)
4. Make sure the dropdown on the left of the aXe developer tools panel says **All issues** or **Violations**
5. Verify the error exists

## Pages Affected
* https://staging.va.gov/track-claims/your-claims/600141546/status

## Possible Fix
Removing the `role="presentation"` from the `<li>` elements will fix the issue:

```html
<ol class="process form-process claim-timeline">
  <li class="process-step list-one section-complete">
    <i class="fa fa-plus claim-timeline-icon"></i>
    <h5 class="section-header">Claim received</h5>
  </li>
  ...
</ol>
```

## WCAG or Vendor Guidance (optional)
* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

---

![Screen Shot 2018-10-19 at 3.29.33 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/9f5171db-1977-4ca9-a635-18c1b0e0cf3a)

**Comment by andreahewitt-odd** [comment]
Like to issue before migration: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14449

**Comment by Mottie** [comment]
Fixed in https://github.com/department-of-veterans-affairs/vets-website/pull/13105

**Comment by 1Copenut** [comment]
Looks good at the listed link in Staging. Moving to closed!
